target/linux/ramips/rt305x/base-files/etc/board.d/01_leds

```diff
@@ -66,6 +66,12 @@ asiarf,awapn2403)
 dlink,dcs-930l-b1)
 	ucidef_set_led_netdev "wifi" "WiFi" "$boardname:blue:wps"
 	;;
+dlink,dir-506l)
+	ucidef_set_led_gpio "lo_bat" "Low battery" "${boardname}:red:status" "10" "0"
+	ucidef_set_led_default "status" "Status" "${boardname}:green:status" "1"
```

Comment:
This should be implemented by default-state in DTS

Reply:
If I do this then it will ON while boot in progress, idea to ON this then device ready.

Reply:
That's what `led-running = &led_status_green;` in aliases will do anyway.